alf/networks/encoding_networks.py

```diff
@@ -230,6 +230,149 @@ def __init__(self,
         super().__init__(nets, input_tensor_spec=input_tensor_spec, name=name)
 
 
+@alf.configurable
+class ImageDeconvNetwork(_Sequential):
```

Comment:
Suggest renaming it to a more informative name differentiating from ``ImageDecodingNetwork``, e.g., ImageDeconvAutoCalcNetwork

Reply:
Changed name to ``AutoShapeImageDeconvNetwork``.